Fix broken autoreconf

Downloaded from upstream git
https://git.xiph.org/?p=theora.git;a=commitdiff;h=28cc6dbd9b2a141df94f60993256a5fca368fa54

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>


From: Tim Terriberry <tterribe@xiph.org>
Date: Fri, 20 May 2011 20:41:50 +0000 (+0000)
Subject: Make autoreconf -i -f work.
X-Git-Url: https://git.xiph.org/?p=theora.git;a=commitdiff_plain;h=28cc6dbd9b2a141df94f60993256a5fca368fa54

Make autoreconf -i -f work.

Patch from David Schleef.

svn path=/trunk/theora/; revision=17990
---

diff --git a/Makefile.am b/Makefile.am
index 89fd753..1783857 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,6 +4,8 @@
 #AUTOMAKE_OPTIONS = foreign 1.7 dist-zip dist-bzip2
 AUTOMAKE_OPTIONS = foreign 1.11 dist-zip dist-xz
 
+ACLOCAL_AMFLAGS=-I m4
+
 if THEORA_ENABLE_EXAMPLES
 EXAMPLES_DIR = examples
 else
diff --git a/configure.ac b/configure.ac
index 1cbec1a..456b603 100644
--- a/configure.ac
+++ b/configure.ac
@@ -68,7 +68,7 @@ AC_LIBTOOL_WIN32_DLL
 AM_PROG_LIBTOOL
 
 dnl Add parameters for aclocal
-AC_SUBST(ACLOCAL_AMFLAGS, "-I m4")
+AC_CONFIG_MACRO_DIR([m4])
 
 dnl Check for doxygen
 AC_ARG_ENABLE([doc],
